contracts/ethereum/contracts/Operator.sol

```diff
@@ -0,0 +1,69 @@
+// SPDX-License-Identifier: GPL
+
+// Copyright 2022 Fluidity Money. All rights reserved. Use of this
+// source code is governed by a GPL-style license that can be found in the
+// LICENSE.md file.
+
+pragma solidity 0.8.11;
+pragma abicoder v2;
+
+import "./Token.sol";
+import "./IUtilityClient.sol";
+
+contract Operator {
+    event NewUtilityClient(address name, address client);
```

Comment:
Clarify the language to eliminate the distinction between "Client" and "Server" if we're not using Utility Mining server